elements/pfe-tabs/src/pfe-tab.scss

```diff
@@ -3,252 +3,376 @@
+  color: pfe-local(Color--focus);
+}
+
+:host([pfe-variant="earth"][on="saturated"]:hover) {
+  color: pfe-local(Color--focus);
+}
+
+:host([pfe-variant="earth"][on="saturated"][aria-selected="true"]) {
+  --pfe-tabs--Color: #{pfe-var(text)};
+}
+
 /// ===========================================================================
-/// EARTH TAB VERTICAL ORIENTATION
+/// WIND TAB VARIANTS ON DARK & SATURATED
 /// ===========================================================================
 
-@media screen and (min-width: #{pfe-breakpoint(md)}) {
-  :host([vertical][pfe-variant="earth"]) {
-    --pfe-tabs__tab--BorderTop: #{pfe-var(ui--border-width)} #{pfe-var(ui--border-style)} #{pfe-var(surface--border)};
+// SATURATED
+:host([on="saturated"][pfe-variant="wind"]) {
+  color: pfe-var(text--on-saturated);
 
-    @include browser-query(ie11) {
-      border-top: 1px solid #{pfe-var(surface--border)};
-    }
+  @include browser-query(ie11) {
+    color: pfe-fetch(text) !important;
+    background-color: transparent;
   }
+}
 
-  :host([vertical][pfe-variant="earth"]:first-of-type) {
-    --pfe-tabs__tab--BorderTop: 0;
+:host([on="saturated"][pfe-variant="wind"][aria-selected="true"]) {
+  --pfe-tabs--Color--focus: #{pfe-var(text--on-saturated)};
+  --pfe-tabs--BorderColor: pfe-var(surface--lightest);
 
-    @include browser-query(ie11) {
-      border-top: 0;
-    }
+  @include browser-query(ie11) {
+    color: pfe-fetch(text) !important;
   }
+}
 
-  :host([vertical][pfe-variant="earth"][aria-selected="true"]) {
-    --pfe-tabs__tab--BorderTop:   #{pfe-var(ui--border-width)} #{pfe-var(ui--border-style)} #{pfe-var(surface--border)};
-    --pfe-tabs__tab--BorderRight: 0;
-    --pfe-tabs__tab--BorderLeft:  #{pfe-var(surface--border-width--heavy)} #{pfe-var(surface--border-style)} #{pfe-local(highlight)};
+:host([on="saturated"][pfe-variant="wind"][aria-selected="false"]) {
+  --pfe-tabs--Color: #{pfe-var(text--muted--on-saturated)};
+}
 
-    @include browser-query(ie11) {
-      border-top: 1px solid #{pfe-var(surface--border)};
-      border-right: 0;
-      border-left: 3px solid #{pfe-local(highlight)};
-    }
+:host([on="saturated"][pfe-variant="wind"][aria-selected="false"]:hover) {
+  border-bottom-color: pfe-local(BorderColor--hover);
+}
+
+:host([on="saturated"][pfe-variant="wind"][aria-selected="true"]:last-of-type) {
+  @include browser-query(ie11) {
+    border-left: 0 !important;
   }
+}
 
-  :host([vertical][pfe-variant="earth"][aria-selected="false"]) {
-    --pfe-tabs__tab--BorderRight:  #{pfe-var(ui--border-width)} #{pfe-var(ui--border-style)} #{pfe-var(surface--border)};
-    --pfe-tabs__tab--BorderBottom: 0;
-    --pfe-tabs__tab--BorderLeft:   #{pfe-var(surface--border-width--heavy)} #{pfe-var(ui--border-style)} #{pfe-var(surface--lighter)};
+// DARK
+ 
+:host([on="dark"][pfe-variant="wind"][aria-selected="true"]) {
+  //--pfe-tabs--Color: #{pfe-var(surface--lightest)};
+  --pfe-tabs--Color--focus: #{pfe-var(surface--lightest)};
```

Comment:
Instead of using a surface color for typography, can we use `text--on-dark`?

Reply:
There's a few examples of surface color being applied to type, same note ^

Reply:
++ thanks, updated